libr/include/r_io.h

```diff
@@ -15,6 +15,9 @@
 
 #define R_IO_UNDOS 64
 
+#define r_io_map_get_from(map) map->itv.addr
```

Comment:
this seems to overlap with your other PR will cause a conflict when trying to merge, so imho is better to wait for the other pr to be merged (just add one test and wait for travis) and then rebase this pr without the dupe stuff

Reply:
really, those are the same commits, afaik git can handle that, you just need to merge the other pr first

Reply:
git creates a conflict if i merge the other pr :P